files/magneticmediajs/info.ini

```diff
@@ -2,4 +2,4 @@ author = "Arnaud Leyder"
 github = "https://github.com/arnaudleyder/magneticmediajs"
 homepage = "https://www.magneticmediajs.com/"
 description = "JavaScript And CSS To Nicely Display Media Content"
-mainfile = "mjs.min.js"
\ No newline at end of file
+mainfile = "js/mjs.min.js"
```

Comment:
ok I have sync my fork with the upstream repo. I have added mainfile files. It is in commit 9d7a4e25a070efd5143b295a8dc870c33b4f6ea7 on my fork. Do I need to do something else?